src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardRetryingRequestResolver.java

```diff
@@ -238,12 +249,12 @@ public void updateData(@NotNull final Map<IToken, Integer> newAssignedRequests,
         this.delays.putAll(newDelays);
     }
 
-    public HashMap<IToken, Integer> getDelays()
+    public Map<IToken<?>, Integer> getDelays()
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1452)